src/frontend/src/pages/CreateChangeRequestPage/CreateProposedSolutionsList.tsx

```diff
@@ -37,7 +33,26 @@ const CreateProposedSolutionsList: React.FC<CreateProposedSolutionsListProps> =
 
   return (
     <>
-      <div className={styles.proposedSolutionsList}>
+      {!isGuest(user.role) && (
+        <Grid container columnSpacing={5}>
```

Comment:
instead of doing this as a grid with grid items, you can do this: `<Box display='flex' justifyContent='space-between'> </Box>` and put the typography/ button inside of that box. Flexbox is pretty useful for responsive ui's check this out if you wanna learn more specifics: https://flexboxfroggy.com/

Reply:
also you'll probably need to shorten the button name to 'add solution' to get that to fit